core/server/data/schema/schema.js

```diff
@@ -172,6 +172,7 @@ module.exports = {
         slug: {type: 'string', maxlength: 150, nullable: false, unique: true},
         secret: {type: 'string', maxlength: 150, nullable: false},
         redirection_uri: {type: 'string', maxlength: 2000, nullable: true},
+        blog_uri: {type: 'string', maxlength: 2000, nullable: true},
```

Comment:
I don't see the need for sorting this value locally? It should be the same as config.url.